### PR TITLE
#2107 Fix python scripts for python3

### LINKED ISSUE #2107: Unit tests does not pass with python3 (mainly due to encoding issues)

### PR DESCRIPTION
**Pull request by rosiereflo**
closes #2107

### CODE REVIEW

**Comment by rosiereflo** [comment]
Need to be tested on centos6 and 7

**Comment by rosiereflo** [comment]
I recompiled on centos6 and 7 and it seems to be ok...

File: util/timeline_filter/timelineFilter.py [hunk 1]
```diff
@@ -204,6 +180,12 @@ def loadOppositeEventsTable():
     return dicOppEvents
 
 def parseXml(filepath, timeline):
+    try:
+        fh = codecs.open(filepath, 'r', encoding='utf-8')
```
Comment (marcochiaramello) [inline util/timeline_filter/timelineFilter.py:184]: it's this additional parsing need? I was thinking that elementree was able to check itself, but maybe I am wrong.

**Review by marcochiaramello — CHANGES_REQUESTED** [review]

File: util/timeline_filter/timelineFilter.py (hunk 1 above)
Comment (rosiereflo) [reply util/timeline_filter/timelineFilter.py:184]: recent versions are able to detect the formating, but not old ones, and we have to be able to run on python2

Comment (marcochiaramello) [reply util/timeline_filter/timelineFilter.py:184]: ok, thx!

**Review by marcochiaramello — APPROVED** [review]

**Review by gautierbureau — APPROVED** [review]

**Pull request merged**